mne/io/brainvision/brainvision.py

```diff
@@ -222,15 +226,13 @@ def _read_segments_c(raw, data, idx, fi, start, stop, cals, mult):
     _mult_cal_one(data, block, idx, cals, mult)
 
 
-def _read_mrk(fname, ignore_marker_types=False):
+def _read_mrk(fname):
```

Comment:
IIUC, `ignore_marker_types` is exposed in the public API of `read_annotations`, passed through to `_read_annotations_brainvision` and then to here. Why are you simply removing it??

Reply:
Because it is not needed here anymore.

Reply:
But by doing this, won't users lose the ability to control whether marker types are included in their annotations? If it's removed here, shouldn't it be removed from the parameters of the upstream functions that call this one? And since one of those is in the public API, shouldn't there be a deprecation period?

I'm actually quite surprised that the tests are passing, since `_read_mrk` is called with `ignore_marker_types` here:

https://github.com/mne-tools/mne-python/blob/afd69afe9ed9b07d48e860537d37fc349def6635/mne/io/brainvision/brainvision.py#L347-L349

which is in turn called from `read_annotations()`:

https://github.com/mne-tools/mne-python/blob/afd69afe9ed9b07d48e860537d37fc349def6635/mne/annotations.py#L1226

...which ought to trigger an unexpected keyword argument.

Reply:
You were probably looking in the wrong branch, since this commit is not part of this PR. I actually removed the `ignore_marker_types` parameter from `_read_mrk()` and instead handle it in the `_read_annotations_brainvision()` function (which calls `_read_mrk()` without the `ignore_marker_types` now).

Reply:
I dunno what happened, but yeah I now see the handling of `ignore_marker_types` elsewhere in the diff. Sorry for the noise